File: src/models/memoTemplate.ts
export type MemoTemplateDetail = {
  id: number;
  slug_name: string;
  name: string;
  content: string;
  is_default: boolean;
  workspace_id: number;
  created_at: string;
  updated_at: string;
};

export type MemoTemplateIndexItem = {
  id: number;
  slug_name: string;
  name: string;
  is_default: boolean;
  created_at: string;
  updated_at: string;
};

File: src-tauri/src/models/memo_template.rs
use serde::{Deserialize, Serialize};

#[derive(Serialize, Deserialize)]
pub struct MemoTemplateDetail {
    pub id: i32,
    pub slug_name: String,
    pub name: String,
    pub content: String,
    pub is_default: bool,
    pub workspace_id: i32,
    pub created_at: String,
    pub updated_at: String,
}

#[derive(Serialize, Deserialize)]
pub struct MemoTemplateIndexItem {
    pub id: i32,
    pub slug_name: String,
    pub name: String,
    pub is_default: bool,
    pub created_at: String,
    pub updated_at: String,
}
